browser/static/js/adjust.js

```diff
@@ -138,8 +138,8 @@ function preCompAdjust() {
   preCompSeg.src = canvas.toDataURL();
 }
 
-// adjust raw further (if needed), composite annotations on top
-function compositeImages() {
+// adjust raw further, pre-compositing (use to draw when labels hidden)
+function preCompRawAdjust() {
```

Comment:
If possible, we should edit this function so that it doesn't assume any global variables.  If this function is called in the wrong order, it may `rawData`, `rawWidth`, `rawHeight`, or `preCompRaw` may have unexpected values.

Instead, I think we can edit this function (and eventually all of the functions) so that they use as few out-of-scope variables as possible.

Reply:
I'd like to address this in a separate PR to refactor `adjust.js`, if that seems appropriate to you.

Reply:
I don't think its best to commit a function that already has a TODO in it. That said, maybe it is appropriate to wait for the `adjust.js` refactor; I don't know what that entails.

The more functions we add that use these out-of-scope variables, the harder it is to effectively write new code, and the more convoluted the frontend logic becomes.

Reply:
Right, many of the image processing functions in `adjust.js` (the ones that load an image, apply some processing steps, then save the adjusted image to the `src` of an `Image` for later use) also use out-of-scope variables. The change in this PR does split one of those functions into two and adds a few more references to `rawData`, `rawWidth` etc that could be fixed in this PR. I'm suggesting that we could address all the `adjust.js` functions with out-of-scope variables in a separate PR intended for that, instead of fixing `preCompRawAdjust` here and getting the rest later.

I'm open to how you would like me to fix the out-of-scope variables, I just wanted to clarify what I meant by an `adjust.js` refactor.